db_script_ddl.sql

```diff
@@ -505,4 +488,14 @@ FROM Elementy INNER JOIN
 	Elementy_Cechy_Slownik ON Elementy_Cechy.ID_Cecha = Elementy_Cechy_Slownik.ID_Cecha INNER JOIN 
 	Elementy_Jednostki ON Elementy_Cechy.ID_Jednostka = Elementy_Jednostki.ID_jednostka
 GO
-
+/*
```

Comment:
nie komentujemy, jeśli czegoś ma nie być to usuwamy

Reply:
usunięto